internal/component/kafka/kafka.go

```diff
@@ -78,6 +84,9 @@ func (k *Kafka) Init(metadata map[string]string) error {
 	config := sarama.NewConfig()
 	config.Version = meta.Version
 	config.Consumer.Offsets.Initial = k.initialOffset
+	if meta.MinFetchBytes > 0 {
+		config.Consumer.Fetch.Min = meta.MinFetchBytes
```

Comment:
does this line imply that a consumer once initialized for bulk cannot be used for normal subscription?

Reply:
Removed any new setting to kafka sarama config.